datalad/distribution/create.py

```diff
@@ -86,14 +85,6 @@ class Create(Interface):
             doc="""enforce creation of a dataset in a non-empty directory""",
             action='store_true'),
         description=dataset_description,
-        add_to_super=add_to_superdataset,
```

Comment:
I failed to grasp the way `initiate_dataset` is called in the crawler. I could manually search upwards after `ds.create()` and save the superdataset, but feels wrong. Could you point me to a place where the, e.g. openfmri, superdataset actually exists at runtime? It seem this is only happening in the pipelines.